src/App.jsx

```diff
@@ -1,9 +1,24 @@
 import { useState } from 'react';
 import './App.css';
 import Home from './pages/Home';
+import { onAuthStateChanged } from 'firebase/auth';
+import { auth } from '../config.js';
+import SignInButton from './SignInButton';
+import SignOutButton from './SignOutButton';
 
 function App() {
   const [count, setCount] = useState(0);
+  const [username, setUserName] = useState('[not logged in]');
+  const [userEmail, setUserEmail] = useState('[not logged in]');
```

Comment:
i will gonna save this information as a context. you might need to fix this once my branch is merged